src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -33,10 +33,10 @@ const Body = BaseForm.extend(
       const sendText = ( this.model.get('primaryMode') === 'sms' )
```

Comment:
we will need to change this one in the future. Can you create a jira for the backlog?
Instead of having one text with "Send code to" and another with "your phone", we should have
"Send code to {0}" and "Send code to your phone"